substrate/frame/im-online/src/mock.rs

```diff
@@ -114,28 +113,7 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
 
 #[derive_impl(frame_system::config_preludes::TestDefaultConfig)]
 impl frame_system::Config for Runtime {
-	type BaseCallFilter = frame_support::traits::Everything;
-	type BlockWeights = ();
-	type BlockLength = ();
-	type DbWeight = ();
-	type RuntimeOrigin = RuntimeOrigin;
-	type Nonce = u64;
```

Comment:
This is u32 in `TestDefaultConfig`

Reply:
Yes but most likely it will have no impact, so it should be fine.